Neos.Flow/Classes/Property/TypeConverter/AbstractTypeConverter.php

```diff
@@ -119,7 +119,7 @@ public function canConvertFrom($source, $targetType)
      * Returns an empty list of sub property names
      *
      * @param mixed $source
-     * @return array<string>
+     * @return array<mixed>
```

Comment:
Is this really better than just `array`?

Reply:
No, but it is what the interface uses: https://github.com/neos/flow-development-collection/blob/50166ec3fa9dc13a92ba6b1babae6f99318d3cc3/Neos.Flow/Classes/Property/TypeConverterInterface.php#L78 🤷‍♂️

Reply:
Hmm. Okay then 🐑